src/components/BasicTable.jsx

```diff
@@ -184,10 +178,10 @@ function BasicTable() {
               order={order}
               orderBy={orderBy}
               onRequestSort={handleRequestSort}
-              rowCount={rows.length}
             />
             <TableBody>
               {stableSort(rows, getComparator(order, orderBy))
+                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
```

Comment:
should use a const to be descriptive on what this is calculating

Reply:
I made the changes necessary and they will reflect on my next PR